src/decode.rs

```diff
@@ -4524,9 +4524,10 @@ pub(crate) unsafe fn rav1d_decode_frame_init(
         let _ = mem::take(&mut f.lf.level);
```

Comment:
We also need to remove this (the `.clear()` effectively).

Reply:
Done!